core/templates/dev/head/editor/exploration_editor.html

```diff
@@ -1,73 +1,80 @@
 {% extends "base.html" %}
 
 {% block maintitle %}
-  Oppia Editor
+Oppia Editor
```

Comment:
(Also, as a side effect, if all lines change, it becomes quite hard to see exactly what changed in the diff.)